docker/main/ngen/ngen_entrypoint.sh

```diff
@@ -48,6 +48,10 @@ init_ngen_executable_paths
 
 # Move to the output dataset mounted directory
 cd ${OUTPUT_DATASET_DIR:?Output dataset directory not defined}
+#Needed for routing
```

Comment:
Is this hard coding intentional? I don't see any other references to `/dmod/dataset/experiment_output` and we should probably stay away from just naming stuff some form of `experiment` - we're still stuck with that in the GUI.

Reply:
The hard coding is intentional. In short, we need to configure something in advance that will line up with something not known until runtime. 

A troute config file needs to include the path to the ngen output (and its own output).  But job worker have datasets mounted in as a directory with the same name as the dataset itself. Output dataset names are based on the job id, and that is dynamically generated (at least usually). 

I realized after setting this up that we don't absolutely have to do this now, but I decided to leave it in. I could take it out, but it's a problem we eventually will have to deal with, in which case, why not at least do this for now?

I also don't object to changing the name of the symlink, if you have a strong preference about that.

Reply:
Hey, there's a ticket, so I'm good.